chart/compass/templates/managed-postgresql-secret.yaml

```diff
@@ -14,6 +14,32 @@ metadata:
   name: compass-postgresql
   namespace: {{ .Release.Namespace }}
 type: Opaque
+data:
```

Comment:
Leftover duplication of secrets?

Reply:
Actually the secret duplication is needed, because tests that are running in kyma-system namespace will need access to those secrets in order for the pods to access the database.

Reply:
ah yes, I didn't see that the namespace was different. Now it makes sense